src/test/scala/org/broadinstitute/dsde/firecloud/service/SubmissionServiceSpec.scala

```diff
@@ -76,8 +85,6 @@ class SubmissionServiceSpec extends FreeSpec with ScalaFutures with ScalatestRou
       "OK response is returned" in {
         Get(localSubmissionIdPath) ~> Cookie(HttpCookie("iPlanetDirectoryPro", token)) ~> sealRoute(routes) ~> check {
           status should equal(OK)
-          val submission = responseAs[SubmissionIngest]
-          submission shouldNot be (None)
         }
```

Comment:
Good changes, I should have defaulted to a simple OK response without proper response classes. Best to avoid them completely until they are stable and/or in a dsde commons library.